QRadar race conditions (#14803)
demisto/content: Packs/QRadar/ReleaseNotes/2_0_27.md

#### Integrations
##### IBM QRadar v3

> - Fixed an issue where configured **Long Running Instance** along with **Mirror Offenses** fetched duplicate incidents. 

Reviewer: ```suggestion
- Fixed an issue where both **Long Running Instance** and **Mirror Offenses** would fetch duplicate incidents. 
```
